### DISCUSSION TITLE
How to efficiently query which data model does the following table belong to?

### DISCUSSION THREAD

**Discussion by jcjgithub** [apache/doris#38207] [A - General / Q&A]
This is too low if it is determined by the way of query construction statement，The underlying doris must specify which model the table belongs to so that the corresponding operation can be performed . Unfortunately, I did not see this in the official documentation

**Comment by ixzc** [comment]
you can use this sql:
`show create table table_name`
for model selection, you can refer to the documentation below:
[https://doris.apache.org/docs/table-design/best-practice/](url)

**Comment by jcjgithub** [comment]
I knew this way.but besides Have any efficiently method? for example,when I need query amount of table model&nbsp; at a time, this way is low. I expect that table shema can return it when I query&nbsp; information_schema.tables or like it.



---Original---
From: ***@***.***&gt;
Date: Tue, Jul 23, 2024 15:11 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [apache/doris] How to efficiently query which data model does thefollowing table belong to? (Discussion #38207)




 
you can use this sql:
 show create table table_name
 for model selection, you can refer to the documentation below:
 https://doris.apache.org/docs/table-design/best-practice/
